src/components/Home.tsx

```diff
@@ -1,80 +1,21 @@
-import { useEffect, useState } from 'react';
-
 import WrapperFavorites from './WrapperFavorites';
-import SmallInfoCard from './SmallInfoCard';
+import WrapperLastSeen from './WrapperLastSeen';
+import WrapperPopular from './WrapperPopular';
 import MovieBanner from './MovieBanner';
-import Container from './Container';
-import Slider from './Slider';
-import Title from './Title';
-import { Movie } from '../models/index';
-
-import api from 'src/service/api';
-
-export type MoviesType = {
-  vote_count: number;
-  vote_average: number;
-  original_title: string;
-  poster_path: string;
-  overview: string;
-  title: string;
-  id: number;
-};
 
 const Home = () => {
-  const [movies, setMovies] = useState<[]>([]);
-  const [history, setHistory] = useState<MoviesType[]>();
-  const [popularMovies, setPopularMovies] = useState<Movie[]>([]);
- 
-  useEffect(() => {
-    api.getPopularQueryList().then(res => {
-      setMovies(res.slice(0, 10));
-      setPopularMovies(res);
-    });
-    api.getDataByIds(api.getHistoryIdList()).then(res => {
-      setHistory(res);
-    });
-  }, []);
-
   return (
     <>
-      <MovieBanner popularMovies={popularMovies} />
+      <MovieBanner />
```

Comment:
I think it is not relevant - movie banner should have popular movies props, check this